app/controllers/merge_requests_controller.rb

```diff
@@ -32,6 +32,11 @@ def update
     end
   end
 
+  def details
+    @merge_request = MergeRequest.find(params[:id])
+    render :details
+  end
+
```

Comment:
Would it make sense for this to be `show` instead of `details` and the url to be `/merge-requests/{id}` ?